catkin_home/src/main_engine/src/actions/go_to.py

```diff
@@ -0,0 +1,11 @@
+from Action import Action
+
+class go_to(Action):
+    def __init__(self, id, action_client_binded, args):
+        Action.__init__(self,id,action_client_binded,args)
+        print("Initializing GoTo Action instance")
+
+    def run(self):
+        print("Setting goal,filling it and contacting action server of go_to")
```

Comment:
this is just a question, are you going to implement this in another PR? This was already implemented in Action.py

Reply:
yes, the version implemented in Action.py was outdated since chapa made some changes to the nav server (name, type of msg, etc...)